spec/web_helpers.rb

```diff
@@ -0,0 +1,20 @@
+def sign_up(full_name: 'Incredible Yev',
+            email: 'incredible@yev.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.